interfaces/apparmor/template.go

```diff
@@ -233,13 +233,15 @@ var defaultTemplate = []byte(`
 
   # Writable home area for this version.
   owner @{HOME}/snap/@{SNAP_NAME}/@{SNAP_REVISION}/** wl,
+  owner @{HOME}/snap/@{SNAP_NAME}/shared/** wl,
```

Comment:
I partially agree with it's more pleasing to label something in a non negative way. However, I do fell that "unversioned" underly a fact that "common" doesn't hilight well enough: you need to ensure data you put there are  forward compatible.

The "unversioned" term will be a little bit more scary to developers for this and it's a good reason: you really need to ensure that data you put there are working with any available version of your application. You don't want non forward-compatible schema of your database for instance (I'm afraid that this directory will be used a lot for databases in particular).
"common" is a little bit more gentle in this regard and don't underline this risk… at least from a non native speaker perspective (so take that with a grain of salt ;))